linux_docker_resources/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN if test ${PLATFORM} = x86; then curl --silent https://packagecloud.io/gpg.ke
 RUN apt-get update && apt-get install -y build-essential ccache cmake pkg-config python3-empy python3-setuptools python3-vcstool
 
 # Install build and test dependencies of ROS 2 packages.
-RUN apt-get update && apt-get install -y clang-format-3.8 cppcheck git pydocstyle pyflakes python3-coverage python3-mock python3-nose python3-pep8 uncrustify
+RUN apt-get update && apt-get install -y clang-format-3.8 cppcheck git python3-coverage python3-mock python3-nose uncrustify
```

Comment:
I think the idea was to install what ever you could from apt, but everything must be installed in the Python script so that OS X and Windows get the Python dependencies.

I'm agnostic as to whether or not we use the apt version of the packages, though I suppose that might be more like a "typical" user's setup.